Checks/ExternalJS.cs

```diff
@@ -159,6 +161,8 @@ private void LoadJS()
             Debug.LogWarning($"Error loading {fileName}\n{jse.Message}");
         }
     }
+    private void startEngine() { 
+    }
```

Comment:
Does this do anything?

Reply:
nope. was gonna add some then i was like fuck it and i forgot to remove it, TODO today